test/SparseMatrixCSR.jl

```diff
@@ -1,4 +1,4 @@
-module SparseMatrixCSRTests
+#module SparseMatrixCSRTests
```

Comment:
Is it needed to comment the module?

Reply:
Oops. That was a typo. I comment it out to faciliate debugging sessions. Now should be already solved.